src/js/manager/data.js

```diff
@@ -39,6 +42,15 @@ var Data = snippet.defineClass(
         return;
       }
 
+      if(this.options.useCache && keyword in this._cachedKeywords) {
```

Comment:
Please add `space` after `if` statement.

Reply:
There might be a slight chance some inputs are properties of `Object`.
So whether use the `Object.prototype.hasOwnProperty` to check keyword or declare `this._cachedKeywords` with `Object.create(null)`.